mipidsi/src/lib.rs

```diff
@@ -250,7 +250,7 @@ where
     /// The combined height of the fixed area must not larger than the
     /// height of the framebuffer height in the default orientation.
     ///
-    /// After the scrolling region is defined the [`set_scroll_offset`] can be
+    /// After the scrolling region is defined the [Self::set_vertical_scroll_offset] can be
```

Comment:
It's common practice to not include `Self` when referring to methods in the same type, which makes the docs easier to read. Method names should also be inside a code span:
```suggestion
    /// After the scrolling region is defined the [`set_vertical_scroll_offset`](Self::set_vertical_scroll_offset) can be
```

Other parts of the docs also don't follow this convention. I think we should go through the docs an adjust all links to use the same style. Do you want to include this change in this PR? Or would you like me to take a look at this, after this is merged?